public/app/features/correlations/useCorrelations.ts

```diff
@@ -9,25 +9,53 @@ import {
   Correlation,
   CreateCorrelationParams,
   CreateCorrelationResponse,
+  GetAllCorrelationsBySourceUIDParams,
+  GetCorrelationsParams,
   RemoveCorrelationParams,
   RemoveCorrelationResponse,
   UpdateCorrelationParams,
   UpdateCorrelationResponse,
 } from './types';
 
+export interface CorrelationsResponse {
+  correlations: Correlation[];
+  page: number;
+  limit: number;
+  totalCount: number;
+}
+
 export interface CorrelationData extends Omit<Correlation, 'sourceUID' | 'targetUID'> {
   source: DataSourceInstanceSettings;
   target: DataSourceInstanceSettings;
 }
 
+export interface CorrelationsData {
+  correlations: CorrelationData[];
+  page?: number;
+  limit?: number;
+  totalCount?: number;
```

Comment:
This is optional because I use this same interface for both paged and non paged correlations data. The only place we had non paged correlations data was getting correlations by sourceUID which wasnt needed anymore, so I removed that along with making these parameters non-optional.